doctrine.rst

```diff
@@ -603,7 +603,8 @@ the :ref:`doctrine-queries` section.
     indicate that something may not be correct. Click on the icon to open the
     Symfony Profiler and see the exact queries that were executed. If you don't
     see the web debug toolbar, install the ``profiler`` :ref:`Symfony pack <symfony-packs>`
-    by running this command: ``composer require --dev symfony/profiler-pack``.
+    by running this command: ``composer require --dev symfony/profiler-pack`` and return a
+    response with at least HTML code with ``<body></body>``.
```

Comment:
You make instead link this page
https://symfony.com/doc/current/profiler.html
its already document (just after the first image)